build.rs

```diff
@@ -0,0 +1,73 @@
+
+extern crate build_const;
+
+pub fn make_table_crc32(poly: u32) -> [u32; 256] {
```

Comment:
I was thinking we could have an internal crate which `build.rs` used and some of the functions were re-exported in `crc32` and `crc64` -- making it non-breaking.